QUEST_JOBSTEP.tsv

```diff
@@ -2788,7 +2788,7 @@ QUEST_JOBSTEP_20150714_002787	Once you decide to become the greatest, there's no
 QUEST_JOBSTEP_20150714_002788	I like you more as I see you more.{nl}Okay. Let's get started.
 QUEST_JOBSTEP_20150714_002789	I am busy personally.. I am sorry.{nl}I heard a report saying that someone saw the symbol of the lost family so I am chasing after it.
 QUEST_JOBSTEP_20150714_002790	This.. is the symbol of Regwyn Family.{nl}Thanks. As I promised, I will teach you new skills.
-QUEST_JOBSTEP_20150714_002791	Pirates have their levels and Cursoirs too.{nl}Maybe the Honeypins at Tenants' farm will be able to locate you exactly.
+QUEST_JOBSTEP_20150714_002791	Pirates have their levels and Corsairs too.{nl}Maybe the Honeypins at Tenants' farm will be able to locate you exactly.
```

Comment:
I believe within the context of the quest, that 
"Maybe the Honeypins at Tenants' farm will be able to locate you exactly."
would make more sense as
"Maybe defeating the Honeypins at Tenants' farm will allow me to determine yours."
"yours" being your level.